python/tvm/relax/frontend/nn/op.py

```diff
@@ -179,6 +179,89 @@ def matmul(a: Tensor, b: Tensor, out_dtype: Optional[str] = None, name: str = "m
     return _wrap_nested(_op.matmul(a._expr, b._expr, out_dtype=out_dtype), name)
 
 
+def conv1d(
+    input: Tensor,
```

Comment:
Although i originally used this name for conv2d as well, I decided to switch to `x` for consistency since thats what every other op does. We should probably do the same here.

Reply:
Great idea. Changed to `x`. Thanks for your review!